modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -13,9 +13,6 @@ The following procedure walks you through the *From Git* option in the *Develope
 
 . In the *Add* view, click *From Git* to see the *Import from git* form.
 +
```

Comment:
@pmacko1 can you please also remove this plus sign? Sorry I should have spotted that during my first review.

Reply:
yeah sure, I didn't notice it as well.
```suggestion
```

Reply:
i love this new github feature, btw :)
ctrl+g or <triplebacktick>suggestion
<suggestion>
<triplebacktick>